backups/zimbra-common.inc.sh

```diff
@@ -468,3 +468,11 @@ function zimbraCreateDataFolder() {
 
   execZimbraCmd cmd | hideReturnedId
 }
+
+function shellQuietPopd() {
+  command popd > /dev/null
```

Comment:
Do you know why it would be better to use `command` in front of popd/pushd rather than to use directly the popd/pushd?

Reply:
Hi @jvaubourg 
It's only in order to run a shell command ignoring any shell functions (like alias, …)